event_created.html

```diff
@@ -11,7 +11,7 @@
 <div class="section width-narrow padding-large bg-{% if page.custom_fields.page_background_color %}{{ page.custom_fields.page_background_color }}{% else %}white{% endif %}{% if page.custom_fields.page_background_image %}-fade{% endif %}">
 	<div class="section-inner bg-white padding-medium">
 		<h1>{% filter ak_text:"event_created_headline" %}You're almost done!{% endfilter %}</h1>
-		<p class="text-large2">{% filter ak_text:"event_created_text_pt1" %}Check your e-mail{% endfilter %} {% if args.email %} ( {{args.email}} ) {% endif %}{% filter ak_text:"event_created_text_pt2" %}and click the link inside it to confirm your event. If you don't see the email, check your spam folder — or email events@350.org.{% endfilter %}</p>
+		<p class="text-large2">{% filter ak_text:"event_created_text_pt1" %}Check your e-mail{% endfilter %} {% if args.email %} ( {{args.email}} ) {% endif %}{% filter ak_text:"event_created_text_pt2" %}and click the link inside it to confirm your event. If you don't see the email, check your spam folder — or email {{ page.custom_fields.event_contact_email_on_confirmation }}.{% endfilter %}</p>

```

Comment:
What happens if this value is not set by the admin? Perhaps we should do a conditional here to check?

Reply:
@cjmabry i think there should be a default value set for this field: https://act.350.org/admin/core/allowedpagefield/event_5Fcontact_5Femail_5Fon_5Fconfirmation/change/?

Reply:
It looks that way now but I could see if someone edited this custom field that not being the case. For safety and future-proof I think a check in code is preferred but not the biggest deal.

Reply:
@cjmabry updated!